csp-guru/datapackage.json

```diff
@@ -1,15 +1,167 @@
+                        "name": "Lon",
+                        "description": "Longitude of the plant"
+                    },
+                    {
+                        "name": "DNI",
+                        "description": "direct normal irradiance  (DNI) [kWh/m²/y] indicates the solar resources"
+                    },
+                    {
+                        "name": "Year_construction_start",
+                        "description": "indicates the year when construction started"
+                    },
+                    {
+                        "name": "Technology",
+                        "description": "distinguishes CSP designs in Trough, Tower, Hybrid, Fresnel "
+                    },
+                    {
+                        "name": "Capacity",
+                        "description": "nominal capacity of the turbine [MW]"
+                    },
+                    {
+                        "name": "Storage_capacity",
+                        "description": " how many hours the turbine can run on stored heat alone [h]"
+                    },
+                    {
+                        "name": "Field_size",
+                        "description": "the mirror area [㎡]"
+                    },
+                    {
+                        "name": "Cooling",
+                        "description": "cooling technology: dry, wet, or hybrid"
+                    },
+                    {
+                        "name": "Expected_generation",
+                        "description": "yearly power output [GWh/a]"
+                    },
+                    {
+                        "name": "Load_factor",
+                        "description": "percentage of maximum yearly power output, increases with storage"
+                    },
+                    {
+                        "name": "Total_cost",
+                        "description": "in thousand local currency"
+                    },
+                    {
+                        "name": "Specific_investment_cost_2015USD",
+                        "description": "investment cost per MW [2015 USD], discounted"
+                    },
+                    {
+                        "name": "Specific_investment_cost_2015EUR",
+                        "description": "investment cost per MW [2015 EURO], discounted"
+                    },
+                    {
+                        "name": "Developer",
+                        "description": "company that developed the project"
+                    },
+                    {
+                        "name": "HCE_manufacturer",
+                        "description": "heat collecting element (HCE) manufacturer"
+                    },
+                    {
+                        "name": "SCA_manufacturer",
+                        "description": "solar collector assemblies (SCA) manufacturer"
+                    },
+                    {
+                        "name": "Financed_through",
```

Comment:
"name": "Financing_mechanism",

Reply:
Sound better and in the future we can change that but for the moment we need to stick to the published dataset and that says `Financed_through `.